src/main/java/com/eternalcode/parcellockers/ParcelCache.java

```diff
@@ -0,0 +1,37 @@
+package com.eternalcode.parcellockers;
+
+import com.eternalcode.parcellockers.parcel.Parcel;
+import com.eternalcode.parcellockers.parcel.ParcelLocker;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+
+public class ParcelCache {
```

Comment:
Imo te cache powinno być w repo

Reply:
Tylko że wtedy te repozytoria nawzajem będą musiały być od siebie zależne, a raczej nie chcę tak robić, przynajmniej tak to rozumiem

Reply:
Rozwiń, trochę nie jestem pewien o co ci chodzi. Możesz zrobić jakiś prosty diagram?

Reply:
![image](https://github.com/EternalCodeTeam/ParcelLockers/assets/77227023/1a0b1396-e6da-4107-8496-4c95320014a2)
Obecnie wygląda to tak